### DISCUSSION TITLE
Add console to the panel

### DISCUSSION THREAD

**Discussion by AZNAVOURD** [Paymenter/Paymenter#500] [Ideas]
Can you add the console of the pterodactly panel directly to paymenter , so that there is only 1 pannel that does everything.

**Comment by GomorrhaDev** [comment]
This will not be implemented l, I can guarantee you that.
But you should take a look at Jexactyl.

**Reply by KyGekk** [reply]
Why not?

**Reply by GomorrhaDev** [reply]
Since Paymenter is not just a billing panel for Pterodactyl, but a solution for many types of servers, Pterodactyl Exclusive functions are rather inappropriate.
In addition, just one console would not be enough; in the long run, the entire range of Pterodactyl functions would have to be integrated into Paymenter, and Paymenter is simply not designed for that.

**Reply by KyGekk** [reply]
Fair Point

**Reply by GomorrhaDev** [reply]
But there is this one:
https://market.paymenter.org/extensions/52/Pterodactyl
Don't know if it works
